po/ru.po

```diff
@@ -1647,28 +1771,64 @@ msgid ""
 "      * It is also extensible with sub command plugins as well (such as [cargo clippy]).\n"
 "  * Read more from the [official Cargo Book]"
 msgstr ""
+"* У Rust стремительный график релизов: новая версия выходит каждые шесть недель.\n"
+"Новые версии поддерживают обратную совместимость со старыми версиями —\n"
+"в дополнение они предоставляют новые функциональные возможности.\n"
+"\n"
+"* Существует три вида релизов: ”stable”, ”beta” и ”nightly”.\n"
+"\n"
+"* Новые функции тестируются на ”nightly”, ”beta” - это то, что становится\n"
+"”stable” каждые шесть недель.\n"
+"\n"
+"* У Rust также есть [редакции]: текущая редакция --- Rust 2021. Предыдущими\n"
```

Comment:
Looks like an mdash-to-be? :)

Reply:
Looks like it

Reply:
I'm not sure if you're aware: `mdbook` turns `---` into an em-dash automatically (and similarly for `--`).